python/options.py

```diff
@@ -0,0 +1,55 @@
+import argparse
```

Comment:
Again, switch from argparser to optparser.  The argparser appears to be included only after python3.2.  The default version of python on slc6 is python2.6.6, on cc7 it is 2.7.5, and on lxplus we are setting it to 2.7.4 in the repository setup.  We are not supporting python 3.X as this would require changes to the other python scripts.